sys-auth/pam_u2f/pam_u2f-1.1.0.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools flag-o-matic pam
+
+DESCRIPTION="PAM module for FIDO2 and U2F keys"
+HOMEPAGE="https://github.com/Yubico/pam-u2f"
+SRC_URI="https://developers.yubico.com/${PN/_/-}/Releases/${P}.tar.gz"
+
+LICENSE="ISC"
```

Comment:
They didn't switch to `ISC`, just added additional sources that are licensed with `ISC`. So, we need to list both.

Reply:
My bad. Thanks for fixup!